### PR TITLE
feat(cli): get/set db config key

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- unify Logto config key/guard definitions in `schemas/src/types/logto-config.ts`
- add `logto db get-key` command to get config values by keys from database
- add `logto db set-key` command to set config value by key to database in a JSON favor

## Testing

image

image

image

image

### CODE REVIEW

File: packages/cli/src/commands/database/key.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  );
+
+  if (invalidKey) {
+    log.error(
+      `Invalid config key ${chalk.red(invalidKey)} found, expected one of ${validKeysDisplay}`
+    );
+  }
+};
+
+export const getKey: CommandModule<unknown, { key: string; keys: string[] }> = {
```
Comment (simeng-li) [inline packages/cli/src/commands/database/key.ts:30]: nit: getByKey, setByKey?

File: packages/cli/src/commands/database/key.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      }),
+  handler: async ({ key, keys }) => {
+    const queryKeys = deduplicate([key, ...keys]);
+    validateKeys(queryKeys);
+
+    const pool = await createPoolFromConfig();
+    const { rows } = await getRowsByKeys(pool, queryKeys);
+    await pool.end();
+
+    console.log(
```
Comment (simeng-li) [inline packages/cli/src/commands/database/key.ts:54]: log.info?

File: packages/cli/src/commands/database/key.ts (hunk 1 above)
Comment (gao-sun) [reply packages/cli/src/commands/database/key.ts:30]: good call. i think it should rename to `get-config`?

File: packages/cli/src/commands/database/key.ts (hunk 2 above)
Comment (gao-sun) [reply packages/cli/src/commands/database/key.ts:54]: yes, will do it later

File: packages/cli/src/commands/database/key.ts (hunk 1 above)
Comment (gao-sun) [reply packages/cli/src/commands/database/key.ts:30]: LOG-4324

File: packages/cli/src/commands/database/key.ts (hunk 2 above)
Comment (gao-sun) [reply packages/cli/src/commands/database/key.ts:54]: LOG-4324

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**